src/middleware/authMiddleware.ts

```diff
@@ -0,0 +1,45 @@
+import jwt from 'jsonwebtoken';
```

Comment:
make pull request i created this file and i have middleware called protected which verify token